src/opencl_DES_bs_b_plug.c

```diff
@@ -15,7 +15,12 @@
 #include "../run/opencl/opencl_DES_hst_dev_shared.h"
 #include "mask_ext.h"
 
-#define CONFIG_FILE 	"$JOHN/opencl/DES_bs_kernel_b_%s.config"
+#if JOHN_SYSTEMWIDE
+#define CONFIG_FILE	JOHN_PRIVATE_HOME "/opencl/DES_bs_kernel_b_%s.config"
+#else
+#define CONFIG_FILE	"$JOHN/opencl/DES_bs_kernel_b_%s.config"
```

Comment:
I guess it's whitespace changes on the `CONFIG_FILE` line here and in other changed files? If so, I hope they're for the better.

Reply:
They were inconsistently made of tabs and/or spaces. All new ones are a single tab between macro name and definition.